backend/server/server.go

```diff
@@ -28,6 +28,7 @@ import (
 	scas "github.com/qiangmzsx/string-adapter/v2"
 	echoSwagger "github.com/swaggo/echo-swagger"
 	"github.com/tmc/grpc-websocket-proxy/wsproxy"
+	"golang.org/x/exp/slices"
```

Comment:
Instead of introduce x/exp package, we can use slice standard library which be added in go not long ago. https://pkg.go.dev/slices

Reply:
from vscode lsp 😤